package.json

```diff
@@ -125,7 +125,7 @@
     "typescript": "^4.8.4"
   },
   "dependencies": {
-    "@inrupt/solid-client": "^1.23.1",
+    "@inrupt/solid-client": "^1.23.3",
```

Comment:
I was hesitant to do this, but it was required for the PR. Any reason why we didn't have this updated already?